README.md

```diff
@@ -11,65 +11,85 @@
 1. [License](#license)
 
 ## Background
-This repository is a collection of transformers to be used along with VDB as a plugin to fetch, transform and persist log events and storage slots of
-specified MCD contracts.
+This repository is a collection of transformers to be used along with VDB as a plugin to fetch, transform and persist log events and storage slots of specified MCD contracts.
 
 ## Install
-### Dependencies
+
+### Docker Setup
+
+This repository provides a docker-compose setup to manage the multiple container-based deployment for you, if at all possible you should use that setup.
```

Comment:
This seems like the right approach, but maybe worth mentioning the need to be sure the resources allocated to Docker are substantial - I think we'll hit lower limits really fast with the ethereum node included